docker/release/cudaq.wheel.Dockerfile

```
@@ -25,7 +25,8 @@ ADD "$workspace" "$destination"
 
 ARG python_version=3.10
 RUN echo "Building MLIR bindings for python${python_version}" \
-    && python${python_version} -m pip install --no-cache-dir numpy \
+    && python${python_version} -m pip uninstall numpy pytest \
+    && python${python_version} -m pip install --no-cache-dir numpy pytest fastapi uvicorn llvmlite \
```

```suggestion
    && python${python_version} -m pip install --no-cache-dir numpy \
```
As far as we had it documented, the other package were only used for tests, and hence should not be needed because this file only produces the wheels. Tests are run separately in a clean environment (i.e. not the build environment). It could be, of course, that for some reason one of these fixes a missing dependency on arm that the llvm build does not properly complain about?

Also, the two commands above don't do anything after removing the unneeded packages. I think you possibly meant to uninstall numpy from the default python version instead (any pip packages we install in devdeps.manylinux.Dockerfile are already uninstalled again there).

---

The uninstall followed by an install is effectively a _forced_ reinstall. `pip` doesn't have a reinstall option.

I don't think this is an issue in the script stack. Having these packages installed sideways can happen and was happening when I was experimenting.